packages/db/schema.ts

```diff
@@ -177,6 +177,40 @@ export const bookmarks = sqliteTable(
   ],
 );
 
+export const bookmarkReminders = sqliteTable(
+  "bookmarkReminders",
+  {
+    id: text("id")
+      .notNull()
+      .primaryKey()
+      .$defaultFn(() => createId()),
+    bookmarkId: text("bookmarkId")
+      .notNull()
+      .unique() // Each bookmark can only have one reminder
+      .references(() => bookmarks.id, { onDelete: "cascade" }),
+    remindAt: integer("remindAt", { mode: "timestamp" }).notNull(),
```

Comment:
Currently, each `bookmark` will only have one `bookmarkReminder`, so we could have just added the `remindAt` field into `bookmark` table. However this was just a decision made to make implementation scope smaller. We could very well allow a bookmark to have many reminders (like google calendar events). In that case this `bookmarkReminders` will make it easy for us to make that transition.

@MohamedBassem any opinion or preference on how we should do with the database schema?